### ISSUE TITLE
Monospaced font is no longer monospaced

### ISSUE THREAD

**Issue by andyparkins** [telegramdesktop/tdesktop#7776]
### Steps to reproduce
1.  Open any chat
2.  Format text as "monospace", or wrap in backticks
3.  Send message

### Expected behaviour
Text should be formatted in a monospaced font.

### Actual behaviour
Text changes font, but it is not monospaced.  When used for discussing code fragments or debug log output or even little ASCII diagrams that makes that text incorrectly formatted.

### Configuration
Debian 10

**Version of Telegram Desktop:** 2.1.1

**Installation source (Linux Only)** - the official website originally but upgraded in-app since then

**Used theme**: default

<details><summary><b>Logs</b>:</summary>
[2020.05.02 09:47:23] Primary screen DPI: 235.975
[2020.05.02 09:47:23] Font: from ':/gui/fonts/DAOpenSansRegular.ttf' loaded 'DAOpenSansRegular'
[2020.05.02 09:47:23] Font: from ':/gui/fonts/DAOpenSansRegularItalic.ttf' loaded 'DAOpenSansRegularItalic'
[2020.05.02 09:47:23] Font: from ':/gui/fonts/DAOpenSansBold.ttf' loaded 'DAOpenSansBold'
[2020.05.02 09:47:23] Font: from ':/gui/fonts/DAOpenSansBoldItalic.ttf' loaded 'DAOpenSansBoldItalic'
[2020.05.02 09:47:23] Font: from ':/gui/fonts/DAOpenSansSemibold.ttf' loaded 'DAOpenSansSemibold'
[2020.05.02 09:47:23] Font: from ':/gui/fonts/DAOpenSansSemiboldItalic.ttf' loaded 'DAOpenSansSemiboldItalic'
</details>

**Comment by andyparkins** [comment]
![Screenshot_20200502_100730](https://user-images.githubusercontent.com/903332/80859998-bcc30a00-8c5c-11ea-916d-bbdd223becfa.png)

Screenshot in case it helps -- demonstrates that characters are not all a single width.  I can't identify by sight which font is actually being used but it has changed in a recent update.

**Comment by ilya-fedin** [comment]
According to your screenshot - it is monospaced
If you don't like it, change you system settings

**Comment by ilya-fedin** [comment]
>  but it has changed in a recent update

Previously Liberation Mono was forced, now your system monospace font is used

**Comment by andyparkins** [comment]
> According to your screenshot - it is monospaced

Not so -- it's a different font; but if it were monospaced, "x" would be the same width as "w" and the numbers would line up... here's the same text in this comment box:

```
1234567890
  34
    56
      78
        90
xxxxxxxxxx
wwwwwwwwww
```

which is what I expect a monospaced font to look like.

**Comment by andyparkins** [comment]
> Previously Liberation Mono was forced, now your system monospace font is used

Ah ... okay.  My system font was set to "Monospaced" and looked monospaced in the preview but I've picked a different font and restarted Telegram.  It now looks like this.

![Screenshot_20200502_102602](https://user-images.githubusercontent.com/903332/80860399-5390c600-8c5f-11ea-8307-979c5b7bb29b.png)

which is great.

Thanks for help.  Whatever the problem was it's obviously not Telegram's fault.  Closing issue.
